code/modules/submaps/submap_job.dm

```diff
@@ -45,18 +43,6 @@
 		owner = _owner
 		..()
 
-/datum/job/submap/is_species_allowed(datum/species/S)
```

Comment:
While we do not have different species, this seems out of scope and should be declared in the pr changelogs to bring it into scope.

Same for most of the clothing items that were removed.

Reply:
Included